python/kserve/kserve/api/watch.py

```diff
@@ -21,7 +21,7 @@
 from ..utils import utils
 
 
-def isvc_watch(name=None, namespace=None, timeout_seconds=600):
+def isvc_watch(name=None, namespace=None, timeout_seconds=600, generation=0):
```

Comment:
The generation field can be fetched from isvc metadata, so I think we do not need to pass `generation` here.